packages/dashmate/configs/migrations.js

```diff
@@ -627,6 +627,9 @@ module.exports = {
 
         config.dashmate.helper.docker.build = systemConfigs.base.dashmate.helper
           .docker.build;
+
+        // eslint-disable-next-line max-len
+        config.platform.drive.tenderdash.genesis.chain_id = systemConfigs.testnet.platform.drive.tenderdash.genesis.chain_id;
```

Comment:
Won't this set all configs to the testnet string, even if the network is not `testnet`?